src/routes/console/project-[project]/overview/platforms/wizard/flutter/step3.svelte

```diff
@@ -19,10 +20,10 @@ client
 <WizardStep>
     <svelte:fragment slot="title">Let's get coding</svelte:fragment>
```

Comment:
```suggestion
    <svelte:fragment slot="title">Initialize SDK</svelte:fragment>
```

Looks like you missed one :P